models/comments.model.js

```diff
@@ -10,24 +11,22 @@ exports.fetchComments = (articleId) => {
     .then((data) => {
         return data.rows;
     })
-    .catch((err) => {
-        return err;
-    })
 }
 
-exports.existenceCheck = (articleId) => {
+exports.saveComment = (newComment) => {
+    const comment = {
+        author: newComment.username, 
+        body: newComment.body, 
+        votes: 0,
+        article_id: newComment.article_id
+    }
+    const commentValues = Object.values(comment);
     return db.query(`
-        SELECT * FROM articles WHERE article_id = $1
-    `, [articleId])
+        INSERT INTO comments(author, body, votes, article_id) 
```

Comment:
not necessary to send votes as our DB will create the votes property with a default value of 0

Reply:
fixed